### PR TITLE
nsd 4.1.2

### PR DESCRIPTION
**Pull request by arhbkb**
This PR updates nsd to version 4.1.2

### CODE REVIEW

**Comment by arhbkb** [comment]
I removed the bottle section. Apologies for this. I only saw later in the documentation that I should leave the bottle section, and let you update it. I'll keep this in mind for future PRs.

**Comment by mikemcquaid** [comment]
> I removed the bottle section. Apologies for this. I only saw later in the documentation that I should leave the bottle section, and let you update it. I'll keep this in mind for future PRs.

Can you re-add it? Thanks!

**Comment by arhbkb** [comment]
On 19 April 2015 at 12:16, Mike McQuaid <notifications@github.com> wrote:

> I removed the bottle section. Apologies for this. I only saw later in the
> documentation that I should leave the bottle section, and let you update
> it. I'll keep this in mind for future PRs.
>
> Can you re-add it? Thanks!
>
Should I create a new PR, or just update the formula and push it?

**Comment by mikemcquaid** [comment]
> just update the formula and push it?

This, thanks :+1:

**Comment by arhbkb** [comment]
Okay, I've added the bottle section back in.

**Comment by arhbkb** [comment]
Would you please like to wait before merging this? Your comments about the PR for knot led me to audit this with --strict, and I can make some improvements to this as well, and give it a proper style.

**Comment by arhbkb** [comment]
Okay, so now the nsd formula complies with the brew style guide fully. Please merge.

File: Library/Formula/nsd.rb [hunk 1]
```diff
@@ -24,12 +22,16 @@ def install
...
+    args << "--enable-ratelimit" if build.with? "ratelimit"
+    args << "--enable-zone-stats" if build.with? "zone-stats"
 
     system "./configure", *args
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    system "#{sbin}/nsd", "-v"
```
Comment (mikemcquaid) [inline Library/Formula/nsd.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Comment (arhbkb) [inline Library/Formula/nsd.rb]: The "-v" option prints the version. nsd doesn't have long options. Other than starting the server, I can't think of any other way to test it. I guess that printing the version will have to do for now.

Comment (mikemcquaid) [inline Library/Formula/nsd.rb]: Could start the server using `fork`?

Comment (arhbkb) [inline Library/Formula/nsd.rb]: I suppose that is possible. How do I get the test block to kill the process after a successful test? Or better still, can I run the daemon in the foreground, and have brew send it a TERM signal?

Just a curiosity: in the past, you've happily just accepted a version update. Why are you insisting on so many more tests now?

Comment (mikemcquaid) [inline Library/Formula/nsd.rb]: I'm not insisting; this test is a bit better than nothing but I just figured if you were doing cleanup you might be interested in doing an even better test. If you aren't: that's OK and I'll pull as-is.

**Comment by asparagui** [comment]
Thanks!

**Pull request closed without merging**